src/app/files.js

```diff
@@ -0,0 +1,119 @@
+'use strict'
+
+var EventManager = require('../lib/eventManager')
+
+function Files (storage) {
+  var event = new EventManager()
+  this.event = event
+  var readonly = {}
+
+  this.exists = function (path) {
+    // NOTE: ignore the config file
+    if (path === '.browser-solidity.json') {
+      return false
+    }
+
+    if (this.isReadOnly(path)) {
```

Comment:
I don't mind. The code currently consistently has `if (this.isReadOnly(path)) {`, but it wouldn't make much of a difference.